wasm-wrappers/src/lib.rs

```diff
@@ -355,6 +355,24 @@ pub fn encode_output_transfer(
     Ok(output.encode())
 }
 
+/// Given a destination address, an amount, token ID and a network type (mainnet, testnet, etc), this function
```

Comment:
Two things:

1. I just noticed that we can specify (in a more clearer manner) that amounts are in decimal. This applies to all functions. So just add after `an amoun` in parenthesis (in decimal as a string).
2. Please mentioned after token ID in parenthesis (in address form)

Reply:
Same applies to all functions

Reply:
amounts are a separate struct Amount which is constructed from atoms.